onnx_chainer/.flexci/run_test.sh

```diff
@@ -11,12 +11,13 @@ if [ -n "${GPU+x}" ]; then
     export DOCKER_RUNTIME_ARG="--runtime=nvidia"
     export PYTEST_ARGS=""
 fi
+
 if [ -z "${ONNX_VER+x}" ]; then export ONNX_VER=""; fi
 
 cat <<EOM >test_script.sh
 set -eux
 
-if [[ "${INSTALL_CUPY}" == "on" ]]; then pip install --pre cupy-cuda101; fi
+if [[ "${INSTALL_CUPY}" == "on" ]]; then pip install --pre 'cupy-cuda101<8.0.0'; fi
```

Comment:
This one is tricky as we don't have easy access to chainer version ...
`python -c 'import chainer; print(chainer.__version__)'` fails here